elements/document/file/nuxeo-file-create-layout.html

```diff
@@ -33,6 +33,14 @@
       required
     >
     </nuxeo-input>
+    <nuxeo-input
```

Comment:
nit: please leave a space between widgets, for consistency with the rest of the file:
```suggestion

    <nuxeo-input
```